src/menu.test.visuals.ts

```diff
@@ -36,6 +37,12 @@ for (const story of stories) {
               element.open = true;
             });
 
+          await page
+            .locator('glide-core-menu glide-core-menu')
+            .evaluate<void, Menu>((element) => {
+              element.open = true;
```

Comment:
This is the `offset` set. I figured it would be good to also include the sub-Menu in the screenshot.

Reply:
Good idea.